doc/size.txt

```diff
@@ -0,0 +1,14 @@
+* _.size(t)
+
+  Returns the size of 't'. Accepts tables and strings.
+  Note that this function will not stop counting when
```

Comment:
I think it'd be better just to say this counts the number of keys in a table, rather than trying to think of it as an array. Maybe also add an example to demonstrate that.

Reply:
> I think it'd be better just to say this counts the number of keys in a table, rather than trying to think of it as an array.

Probably, yes.

> Maybe also add an example to demonstrate that.

```Lua
> _.size({1, 2, nil, 3})
3
```
I do have this example, does that not do the trick?

Reply:
Better add a use showcasing non-number keys as well.

Reply:
Sorry - I didn't see this when you posted it yesterday and then forgot to post today. Yeah, my complaint with that example is that it's still "array like"\*. Something like `{ a = 1, b = 2 }` would be fine.


\*<sub>It's sufficiently dense it'll store all items in the array part.</sub>